tests/garage/torch/algos/test_loss_function_utils.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import tensorflow as tf
+import torch
+
+import garage.tf.misc.tensor_utils as tf_utils
+import garage.torch.algos.loss_function_utils as torch_loss_utils
+from tests.fixtures import TfGraphTestCase
+
+
+class TestLossFunctionUtils(TfGraphTestCase):
+
+    def test_compute_advantages(self):
+        discount = 0.99
+        gae_lambda = 1.
```

Comment:
test this with something other than 1, since this `gae_lambda` won't produce a meaningful decay.

Reply:
fixed